db-migrations/27_deployment_mode.up.sql

```diff
@@ -0,0 +1,2 @@
+CREATE TYPE deployment_mode as ENUM ('raw_deployment', 'serverless');
+ALTER TABLE version_endpoints ADD COLUMN deployment_mode deployment_mode NOT NULL default 'serverless';
```

Comment:
does it make spike load on db when we do migration?

Reply:
Could be, but I think the impact/risk is not that high